app/jobs/concerns/resource_request_jobs/processing_helpers.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module ResourceRequestJobs
+  module ProcessingHelpers
+    extend ActiveSupport::Concern
+
+    def with_shared_error_handling(resource_request_id)
+      yield
+    rescue Faraday::ConnectionFailed
+      Rails.logger.error("Unable to connect to Hyacinth, so #{self.class.name} for resource request #{resource_request_id} failed.")
```

Comment:
Should the error message be logged?

Also, can Faraday::TimeoutError be thrown/raised?

Reply:
Yes on both counts.  I'll update the code, thanks!